src/main/java/org/folio/dataexp/service/export/strategies/AuthorityExportStrategy.java

```diff
@@ -0,0 +1,66 @@
+package org.folio.dataexp.service.export.strategies;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.dataexp.domain.dto.MappingProfile;
+import org.folio.dataexp.domain.entity.MarcRecordEntity;
+import org.folio.dataexp.repository.MarcAuthorityRecordRepository;
+import org.folio.dataexp.service.ConsortiaService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Log4j2
+@Component
+@RequiredArgsConstructor
+public class AuthorityExportStrategy extends AbstractExportStrategy {
+
+  private final RuleFactory ruleFactory;
+  private final ConsortiaService consortiaService;
+  private final MarcAuthorityRecordRepository marcAuthorityRecordRepository;
+
+  @Value("${myuniversity}")
+  private String tenantId;
```

Comment:
tenantId will be not found in multiple tenant environment, to get tenant information please use FolioExecutionContext bean.

Reply:
Good catch, thank you.